README.adoc

```diff
@@ -95,9 +115,23 @@ Because the JAX-RS framework handles basic API generation for JAX-RS annotations
 tree will be generated from the `inventory` service. You can use this tree as a starting point and
 augment it with annotations and code to produce a complete OpenAPI document.
 
-Now, point to the {openapi-url}[{openapi-url}^] URL to see the generated OpenAPI tree. You can also point to the
-{openapi-url}/ui[{openapi-url}/ui^] URL for a more interactive view of the APIs.
 
+ifndef::cloud-hosted[]
+Now, point to the {openapi-url}[{openapi-url}^] URL to see the generated OpenAPI tree. 
```

Comment:
Now, point to the {openapi-url}[{openapi-url}^] URL 

--->
Now, visit the {openapi-url}[{openapi-url}^] URL